docs/guides/marketing/integrating-with-contentful.md

```diff
@@ -6,115 +6,60 @@ sidebar_position: 6
 
 It is common for marketing and content teams to want an A/B testing solution with a point-and-click visual editor to make straightforward copy or image optimizations. However, these solutions most often rely on blocking scripts that cause performance degradations—slowing your site down by 200-500 milliseconds in best-case scenarios. You could avoid these performance issues by fully implementing feature flags for marketing experiments—but this will mean marketing has to rely on engineering for every test they want to run.
 
-Using a headless CMS like Contentful, and a lightweight, robust feature flagging SDK, like Eppo, allows the best of both worlds: an easy way for marketing teams to run tests without sacrificing performance. In this tutorial, we will walk through how to set up Eppo’s Node SDK so Marketing teams can run their own experiments by updating [Content](https://www.contentful.com/help/content-model-and-content-type/) defined in Contentful and feature flags in Eppo. 
+Using a headless CMS like Contentful, and a lightweight, robust feature flagging SDK, like Eppo, allows the best of both worlds: an easy way for marketing teams to run tests without sacrificing performance. In this tutorial, we will walk through how to set up Eppo’s Node SDK so Marketing teams can run their own experiments by updating [Content](https://www.contentful.com/help/content-model-and-content-type/) defined in Contentful and feature flags in Eppo.
 
+## Installing Eppo's Contentful marketplace app
 
-## Setting up Contentful
+Eppo now supports seamless integration with Contentful via the Eppo Contentful marketplace app. This app allows you to create experiements directly within a Contentful blog post.
 
-For this example, imagine we are running an experiment on the Eppo homepage. There are a few places where we might want to experiment on this page:
+To get started, search for "Eppo" on Contentful's marketplace. To navigate to the marketplace, click on **Apps** from Contentful's navigation menu, then click **Marketplace**.
```

Comment:
we could probably include a link to the actual app page here once it's live. just leaving as a reminder for our future selves